src/Foundation/Connect/code/Builders/Products/BaseProductBuilder.cs

```diff
@@ -61,7 +61,7 @@ protected TEntity BuildWithoutPrices<TEntity>(Item source) where TEntity : BaseP
             var entity = new TEntity();
             this.Initialize(entity, source);
             this.SetCatalogName(entity);
-
+            entity.StockStatusName = "InStock";
```

Comment:
Is everything fail without this line?

Reply:
Yes, add to cart doesn't work without it.